src/Reflector/ClassReflector.php

```diff
@@ -14,12 +18,27 @@ class ClassReflector implements Reflector
      */
     private $reflector;
 
+    /**
+     * @param SourceLocator $sourceLocator
+     */
     public function __construct(SourceLocator $sourceLocator)
     {
         $this->reflector = new GenericReflector($sourceLocator);
     }
 
     /**
+     * @return self
+     */
+    public static function buildDefaultReflector()
```

Comment:
@asgrim it would extend `AggregateSourceLocator` without any other difference with it. I think the factory approach is fine: maybe move it to a separate class? Couldn't find any though